spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j/AmqpAppender.java

```diff
@@ -527,12 +528,6 @@ public void run() {
 						amqpProps.setMessageId(UUID.randomUUID().toString());
 					}
 
-					// Set applicationId, if we're using one
-					if (null != applicationId) {
-						amqpProps.setAppId(applicationId);
```

Comment:
Not sure yet that your `replace` solution is good one, but we shouldn't remove AMQP property anyway